Tests/Integration/DSCResources/DotNetFramework.integration.tests.ps1

```diff
@@ -35,7 +35,7 @@ try
                     $hasAllSettings = $true
                     $dscXml = @($dscXml.DISASTIG.RegistryRule.Rule)
                     $dscMof = $instances |
-                        Where-Object {$PSItem.ResourceID -match "\[Registry\]"}
+                        Where-Object {$PSItem.ResourceID -match '\[Registry[a-zA-Z]*\]'}
```

Comment:
It's probably best we use \[Registry\]|\[RegistryPolicyFile\] for the regex.  It's best to be explicit as possible when it comes to regex.  In the future a resource could be added that starts with "Registry" and could be matched when it shouldn't and now we have to update the tests.  Plus "\[Registry\]|\[RegistryPolicyFile\]" is easier to read than "[Registry.*]".

Reply:
That makes sense